testing/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 3.15)
+
+set(SOURCE_FILES 
```

Comment:
don't use variable just put files in the add_executable

Reply:
Why? Until there is no lib it is easier to divide the sources by variables. Or do I miss something?